src/darwin/Framework/CHIP/templates/tests/helper.js

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+function asTestIndex(index)
```

Comment:
Please document how this is computed and why it's maintainable? Is does seem to expect very specific, non-obvious, test configurations

Reply:
I have added some comment about this method. Please note that while it is here since a while and this PR just moves it, it will also be remove (hopefully soon...) once I'm done moving `src/darwin/Framework` cluster test under `examples/chip-tool-darwin` which will not have the same expectation than the XCTest framework.